src/testUtil/TestCaseRecorder.ts

```diff
@@ -47,6 +47,12 @@ interface RecordTestCaseCommandArg {
    * Whether to flash a background for calibrating a video recording
    */
   showCalibrationDisplay?: boolean;
+
+  /**
+   * Whether we should record a test which yields an error
+   */
+
+  isErrorTest?: boolean;
```

Comment:
I'd be tempted to call this one `recordErrors`, and update the docstring, because the user could start recording using this option, and then record some regular tests in addition to some error tests.  I could see that being useful for capturing where a scope type should and shouldn't match

Also, we usually don't include empty line between field and docstring